### DISCUSSION TITLE
约定式路由的动态路由嵌套问题

### DISCUSSION THREAD

**Discussion by weeptao** [umijs/umi#11541] [Q&A]
约定式路由嵌套
/pages/my
pages
   --my
   |    --index.js
   |--my.js
index和my可以同时展示

约定式路由的动态路由嵌套
但是动态路由
pages/1
   --$my
   |    --index.js
   |--$my.js
只展示$my.js的内容，我想两个都展示，有没有什么方法

**Comment by fz6m** [comment]
动态路由嵌套的时候子路由用从 umi 导入的 `<Outlet />` 展示子级路由内容，或者用 [`wrappers`](https://umijs.org/docs/guides/routes#wrappers) 。

描述里看不清晰，如有问题，需要给一个最小复现。

**Comment by xiaohuoni** [comment]
你想要的是 /index -> index 其他的到 /xxxx -> $my 。不觉得很怪吗？你自己在 $my 里面判断传参数是 index 的时候，展示 index 页面吧

**Comment by weeptao** [comment]
抱歉，我知道了，可能一时糊涂了。问题已解决😊

**Comment by wells974** [comment]
动态路由能和wrappers搭配吗，我试了下好像不生效？ @fz6m @xiaohuoni
